config/routes.rb

```diff
@@ -179,7 +179,10 @@
                 module: 'commodities'
     end
   end
-  get '/find_commodity', to: 'find_commodities#show'
```

Comment:
@alexdesi They are not directly equivalent

* the `resource` line *is* effectively equivalent to a single `GET` line _but_ is the favoured approach within the rails framework, I think because using resource makes following `resource` conventions the default pattern if this is ever gets extended for other actions
* the `resolve` line is the solution for rails defaulting to plural routes rather than singular routes (see https://guides.rubyonrails.org/routing.html#singular-resources)
  * when given a `FindCommodity` model instance, by default rails would look for a route of `/find_commodities/:id`
  * Because this is singular though there is no `:id` and we instead want it to just use `find_commodity_path`